### PR TITLE
Script to extract changelogs from prs

### PR DESCRIPTION
**Pull request by newhoggy**
Example run:

```
$ ../cardano-updates/scripts/download-prs input-output-hk/cardano-api
$ ../cardano-updates/scripts/generate-pr-changelogs input-output-hk/cardano-api cardano-api 2beb008a7f1ce87874b176cbfce44fc7ef7ad74d..cba5cee08d93f3a463e5d0a0a78223496fe382b2
- Add new `checkVrfFilePermissions` function
  (feature; compatible)
  [PR 11](https://github.com/input-output-hk/cardano-api/pull/11)

- <missing changelog>
  [PR 7](https://github.com/input-output-hk/cardano-api/pull/7)

- <missing changelog>
  [PR 1](https://github.com/input-output-hk/cardano-api/pull/1)

- <missing changelog>
  [PR 3](https://github.com/input-output-hk/cardano-api/pull/3)

- <missing changelog>
  [PR 2](https://github.com/input-output-hk/cardano-api/pull/2)
```

### CODE REVIEW

File: scripts/merged-prs/download.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+fi
+
+repository="$1"
+start_date="$("$date_cmd" -u -d "$2" +"%Y-%m-%dT%H:%M:%SZ")"
+end_date="$("$date_cmd" -u -d "$3" +"%Y-%m-%dT%H:%M:%SZ")"
+
+mkdir -p merged-prs
+
+gh pr list --repo "$repository" \
+  -L 1000 \
```
Comment (carbolymer) [inline scripts/merged-prs/download.sh:22]: Will 1000 be enough?

File: scripts/generate-pr-changelogs
Comment (carbolymer) [inline scripts/generate-pr-changelogs:1]: no `sh` extension?

**Review by carbolymer — APPROVED** [review]
README.md requires an update

File: scripts/merged-prs/distribute.sh [hunk 2]
```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
```
Comment (Jimbo4350) [inline scripts/merged-prs/distribute.sh:2]: There are minor differences between this script and `scripts/distribute-merged-prs.sh`. Why not just use one?

File: scripts/merged-prs/download.sh [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+fi
+
+if [[ "$(uname -s)" == "Darwin" ]]; then
+  date_cmd=gdate
+else
+  date_cmd=date
+fi
+
+repository="$1"
+start_date="$("$date_cmd" -u -d "$2" +"%Y-%m-%dT%H:%M:%SZ")"
```
Comment (Jimbo4350) [inline scripts/merged-prs/download.sh:16]: Likewise with `scripts/download-prs.sh`. Why duplicate the scripts?

**Review by Jimbo4350 — COMMENTED** [review]
There is duplication we can avoid

File: scripts/merged-prs/download.sh (hunk 1 above)
Comment (newhoggy) [reply scripts/merged-prs/download.sh]: Now get's the total number of PRs ever created.

**Pull request merged**